models/user.js

```diff
@@ -24,11 +24,11 @@ const UserSchema = new Schema({
   location: String,
   bio: String,
   bestScores: {
-    veryEasy: Number,
-    easy: Number,
-    medium: Number,
-    hard: Number,
-    veryHard: Number,
+    1: Number,
+    2: Number,
+    3: Number,
+    4: Number,
+    8: Number,
```

Comment:
The key names are in numbers but this one doesn't follow the numeral order?

Reply:
Woops, thats a typo, thanks for pointing it out.